docs/platforms/flutter/integrations/app-start-instrumentation.mdx

```diff
@@ -10,13 +10,28 @@ categories:
 
 Sentry's app start instrumentation provides insight into how long your application takes to launch.
 
+<Note>
+
+App start instrumentation is available on **iOS** and **Android**.
+
+</Note>
+
 ## Instrumentation Behaviour
 
 Before diving into the configuration, it's important to understand how app start instrumentation behaves:
 
-- The SDK attaches the app start metrics to the **first run transaction**.
-- It tracks the length of time from the **earliest native process initialization** until the very first [PostFrameCallback](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/addPostFrameCallback.html) is triggered.
-- The SDK differentiates between a cold and a warm start, but doesn't track hot starts/resumes.
+App start instrumentation tracks the duration between the earliest native process initialization and the first frame rendered (as reported by [addTimingsCallback](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/addTimingsCallback.html)). Once the app start is processed, we remove the callback to avoid additional overhead.
+
+When the SDK receives the start and end times of the app launch, we:
```

Comment:
similar to above, avoid "we" here unless the user should do something - please check the rest of the PR for similar occurrences